docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,791 @@ thor:
+        when establishing trust between components in separate HPCC clusters. 
+        </para>
 
-      <sect3 id="CV_CrossOriginRes">
-        <title>Cross Origin Resource Handling</title>
+        <para>To enable TLS for component make these configuration changes:</para>
+        
+        <para><itemizedlist>
+          <listitem>
+            <para>Set the <emphasis>certificates.enabled</emphasis> property to 
+            <emphasis>true</emphasis></para>
+          </listitem>
+          <listitem>
+            <para>Set the <emphasis>enabled</emphasis> property to <emphasis>true</emphasis>
+            for the specific issuer you want to use: local, public or remote</para>
+          </listitem>
+          <listitem>
+            <para>Set the component's <emphasis>visibility</emphasis> property to the value that
+            assigns it the issuer you want to use: `cluster` visibility for `local` issuer, `local` or `global` visibility
+            for the `public` issuer.</para>
+          </listitem>
+          <listitem>
+            <para>To use the `remote` issuer, set the component's `visibility` property to
+            `local` or `global`, and it's `trustClients` array to the list of remote
+            trusted clients.</para>
+          </listitem>
+        </itemizedlist></para>
 
-        <para>Cross-origin resource sharing (CORS) is a mechanism for
-        integrating applications in different domains. CORS defines how client
-        web applications in one domain can interact with resources in another
-        domain. You can configure CORS support settings in the ESP section of
-        the values.yaml file as illustrated below:</para>
+        <note>
+          <para>Although an effort was made to standardize on the term TLS
+          throughout this section, there may be references to the older SSL
+          standard it replaced. Any SSL references should be taken as
+          colloquial or legacy nomencalature that actually refers to a TLS
+          implementation.</para>
+        </note>
 
-        <programlisting>esp:
-- name: eclwatch
-  application: eclwatch
-  auth: ldap
-  replicas: 1
-  # The following 'corsAllowed' section is used to configure CORS support
-  #   origin - the origin to support CORS requests from
-  #   headers - the headers to allow for the given origin via CORS
-  #   methods - the HTTP methods to allow for the given origin via CORS
-  #
-  corsAllowed:
-  # origin starting with https will only allow https CORS
-  - origin: https://*.example2.com
-    headers:
-    - "X-Custom-Header"
-    methods:
-    - "GET"
-  # origin starting with http will allow http or https CORS
-  - origin: http://www.example.com
-    headers:
-    - "*"
-    methods:
-    - "GET"
-    - "POST" </programlisting>
-      </sect3>
-    </sect2>
+        <para>The following examples are to exhibit basic functionality in a 
+        testing environment- production deployments may require different
```

Comment:
Suggestion: the use of the hyphen here.  I think a comma or semi-colon(?) might be more appropriate - e.g.: The following examples demonstrate functionality in non-production environments. Production deployments may require different settings.